bundles.config.js

```diff
@@ -222,6 +222,7 @@ exports.extensionBundles = [
     options: {hasCss: true},
     type: TYPES.MISC,
   },
+  {name: 'amp-skimlinks', version: '0.1', type: TYPES.AD},
```

Comment:
more like `TYPES.MISC`

this is for code bundling. this extension is not using any ads code.